### PR TITLE
Create build-yocto.yml

### PR DESCRIPTION
**Pull request by matinlotfali**

### CODE REVIEW

File: Build/run.sh [hunk 1]
```diff
@@ -11,6 +11,7 @@ usage() {
 #OUTDIR is bind mopunted and will contain the compiled output from the container
 OUTDIR='output'
 CONTNAME="$(whoami)-rzv2l_vlp_v3.0.0"
+test -t 1 && USE_TTY="-it"
```
Comment (mistyron) [inline Build/run.sh:14]: Can you please explain this, what does this do?

**Review by mistyron — COMMENTED** [review]
please reply to comment

File: Build/run.sh (hunk 1 above)
Comment (matinlotfali) [reply Build/run.sh:14]: I noticed that GitHub actions are not using a TTY device when running workflows. This results in errors when docker is executed with the parameter `-it`.
So with this command, I investigate if TTY device is being used.

Comment (mistyron) [reply Build/run.sh:14]: Thank you!

**Review by mistyron — APPROVED** [review]
LGTM

**Pull request merged**